Islandora/islandora_solr_search
# Add title attribute to facet labels

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2471)

# What does this Pull Request do?

Adds the value of the facet label to the "title" attribute in the link HTML, which allows the browser to display the full text on mouseover when the value is truncated by CSS.

# What's new?

Adds the facet label value to the `title` attribute in its link.

e.g. Before: 
`<a rel="nofollow" href="/islandora/search?type=dismax&amp;islandora_solr_search_navigation=0&amp;f%5B0%5D=mods_accessCondition_use_and_reproduction_ms%3A%22http%5C%3A%5C/%5C/rightsstatements.org%5C/vocab%5C/InC%5C/1.0%5C/%22">http://rightsstatements.org/vocab/InC/1.0/</a>      <span class="count">(24)</span>`

After:
`<a rel="nofollow" title="http://rightsstatements.org/vocab/InC/1.0/" href="/islandora/search?type=dismax&amp;islandora_solr_search_navigation=0&amp;f%5B0%5D=mods_accessCondition_use_and_reproduction_ms%3A%22http%5C%3A%5C/%5C/rightsstatements.org%5C/vocab%5C/InC%5C/1.0%5C/%22">http://rightsstatements.org/vocab/InC/1.0/</a>      <span class="count">(24)</span>`

# How should this be tested?

* Add some facet fields
* Do a search
* Mouse over your facet links and note that there's no mouseover text
* View Source and find those links; no `title` attribute
* Check out this branch and repeat search
* Observe mouseover text appears
* View Source and observe the `title` attribute is present
* Look for any weird behaviour (it should be fine, but good to check)

# Additional Notes:

Just a display thing. If for some reason you DON'T want your facets' values to be visible on mouseover, you could always hide it with CSS - but without this tweak to the code, there's no way to turn ON this feature.

# Interested parties
@Islandora/7-x-1-x-committers
